src/manifold/include/manifold.h

```diff
@@ -249,6 +249,16 @@ class Manifold {
   static Manifold Hull(const std::vector<glm::vec3>& pts);
   ///@}
 
+  /** @name Voronoi Functions
+   */
+  ///@{
+  std::vector<Manifold> Fracture(const std::vector<glm::dvec3>& pts,
```

Comment:
Is fracture really meant to be a public function, or is it just a means to ConvexDecomposition?

Reply:
I think it's worth making it public because it's a non-trival operation that generalizes splitting a singular cutting plane by limiting the influence of each half-space.  This can help make modelling cuts and edits more "local".   Like, if you wanted to take a complicated, winding assembly, and break it apart into multiple pieces, it's nice to be able to make cuts with a localized area of effect to avoid mangling the model far from the cut.

Also, it's useful for people looking to implement their own approximate decompositions based on their particular speed and accuracy needs.

And it's good for modelling lattices and packings, which is one of the primary things Voro++ is used for... it could even help replicate NTopology lattice generation functionality if each cell is contracted and then subtracted from a solid region.